scripts/quests/outlands/Black_Market.lua

```diff
@@ -0,0 +1,139 @@
+-----------------------------------
+-- Black Market
+-----------------------------------
+-- Log ID: 5, Quest ID: 130
+-----------------------------------
+local quest = Quest:new(xi.questLog.OUTLANDS, xi.quest.id.outlands.BLACK_MARKET)
+
+quest.reward =
+{
+    title = xi.title.BLACK_MARKETEER,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE
+        end,
+
+        [xi.zone.NORG] =
+        {
+            ['Muzaffar'] = quest:progressEvent(15),
+
+            onEventFinish =
+            {
+                [15] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED
+        end,
+
+        [xi.zone.NORG] =
+        {
+            ['Muzaffar'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:event(16)
+                end,
+
+                onTrade = function(player, npc, trade)
```

Comment:
Minor thing of preference, but I like to sort these functions alphabetically, so onTrade first.